content/en/user-guide/aws/security-token-service/index.md

```diff
@@ -0,0 +1,157 @@
+
+{{< command >}}
+$ awslocal iam create-user \
+    --user-name localstack-user
+{{< /command >}}
+
+You can generate access keys for the IAM user using the [`CreateAccessKey`](https://docs.aws.amazon.com/STS/latest/APIReference/API_CreateAccessKey.html) API. Run the following command to create an access key for the IAM user:
+
+{{< command >}}
+$ awslocal iam create-access-key \
+    --user-name localstack-user
+{{< /command >}}
+
+The following output would be retrieved:
+
+```bash
+{
+    "AccessKey": {
+        "UserName": "localstack-user",
+        "AccessKeyId": "ACCESS_KEY_ID",
+        "Status": "Active",
+        "SecretAccessKey": "SECRET_ACCESS_KEY",
+        "CreateDate": "2023-08-24T17:16:16Z"
+    }
+}
+```
+
+### Create an IAM Role
+
+You can now create an IAM Role, named `localstack-role`, using the [`CreateRole`](https://docs.aws.amazon.com/STS/latest/APIReference/API_CreateRole.html) API. Run the following command to create the IAM Role:
+
+{{< command >}}
+$ awslocal iam create-role \
+    --role-name localstack-role \
+    --assume-role-policy-document '{"Version":"2012-10-17","Statement":[{"Effect":"Allow","Principal":{"AWS":"arn:aws:iam::000000000000:root"},"Action":"sts:AssumeRole"}]}'
+{{< /command >}}
+
+The following output would be retrieved:
+
+```bash
+{
+    "Role": {
+        "Path": "/",
+        "RoleName": "localstack-role",
+        "RoleId": "AROAQAAAAAAAEDP262HSR",
+        "Arn": "arn:aws:iam::000000000000:role/localstack-role",
+        "CreateDate": "2023-08-24T17:17:13.632000Z",
+        "AssumeRolePolicyDocument": {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Principal": {
+                        "AWS": "arn:aws:iam::000000000000:root"
+                    },
+                    "Action": "sts:AssumeRole"
+                }
+            ]
+        }
+    }
+}
+```
+
+You can attach the policy to the IAM role using the [`AttachRolePolicy`](https://docs.aws.amazon.com/STS/latest/APIReference/API_AttachRolePolicy.html) API. Run the following command to attach the policy to the IAM role:
+
+{{< command >}}
+$ awslocal iam attach-role-policy \
+    --role-name localstack-role \
+    --policy-arn arn:aws:iam::aws:policy/AdministratorAccess
+{{< /command >}}
+
+### Assume an IAM Role
+
+You can assume an IAM Role using the [`AssumeRole`](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html) API. Run the following command to assume the IAM Role:
+
+{{< command >}}
+$ awslocal sts assume-role \
+    --role-arn arn:aws:iam::000000000000:role/localstack-role \
+    --role-session-name localstack-session
+{{< /command >}}
+
+The following output would be retrieved:
+
+```bash
+{
+    "Credentials": {
+        "AccessKeyId": "<ACCESS_KEY_ID>,
+        "SecretAccessKey": "SECRET_ACCESS_KEY",
```

Comment:
STS assume role should return a `"SessionToken": ` field, both on AWS and on LocalStack.